cc/algorithms/partition-selection-strategies_test.cc

```diff
@@ -0,0 +1,373 @@
+//
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "algorithms/partition-selection-strategies.h"
+#include "algorithms/numerical-mechanisms-testing.h"
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace differential_privacy {
+namespace {
+
+using testing::_;
+using testing::Eq;
+using testing::DoubleNear;
+using testing::DoubleEq;
+using testing::MatchesRegex;
+
+//PreaggregationPartitionSelection Tests
+
+TEST(PartitionSelectionStrategiesTest, PreaggPartitionSelectionUnsetEpsilon) {
```

Comment:
Very nit: You can call the test suite "PartitionSelectionTest" to reduce line length.